generator/src/lib.rs

```diff
@@ -2374,12 +2377,23 @@ pub fn generate_constant<'a>(
     let c = Constant::from_constant(constant);
     let name = constant_name(&constant.name);
     let ident = format_ident!("{}", name);
+
+    let notation = constant.notation().map(|n| {
+        let lowercase = n.to_lowercase();
+        if lowercase.contains("backwards") || lowercase.starts_with("alias") {
+            quote!(#[deprecated = #n])
+        } else {
+            quote!(#[doc = #n])
+        }
+    });
```

Comment:
Some duplicated code, perhaps we should move this as a default-implementation to `trait ConstantExt`.

Reply:
Did we ever work out what was going on with @MaikKlein's generator rewrite?

Reply:
There are no direct plans to finish it that I'm aware of. I haven't personally looked through it enough to see if I/we could continue on and finish it, nor how far along it is in the first place.